packages/plugin-test-support/package.json

```diff
@@ -41,22 +41,23 @@
     "test:unit": "exit 0"
   },
   "dependencies": {
-    "@appium/types": "^0.26.0",
+    "@appium/types": "^1.0.0-rc.0",
     "get-port": "5.1.1",
     "log-symbols": "4.1.0",
     "source-map-support": "0.5.21",
     "teen_process": "2.3.3"
   },
   "peerDependencies": {
-    "appium": "^2.0.0-beta.43 || ^3.0.0-beta.0",
+    "appium": "^2.0.0-beta.43 || ^3.0.0-rc.1",
```

Comment:
I'd say we keep `^3.0.0-beta.0` everywhere for better compatibility and remove 2.0.0

Reply:
Appium 2 is not maintained anyway. I don't see a point in keeping it after bumping major versions of corresponding packages